### PR TITLE
Qibo blog contribution

### PR DESCRIPTION
**Pull request by francescsabater**

### CODE REVIEW

File: src/content/blog/adding_qibo.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Contributing to Mitiq has truly been a rewarding journey. It has provided me with a profound understanding of the processes involved in quantum computing development and open-source contribution. I would encorage anyone who has an idea that might improve Mitiq to try and contribute. Before any contribution is made, rigorous testing ensures that every aspect functions seamlessly, so there is no reason to be afraid of messing up. Encountering failures during these tests is not uncommon; in my case some tests  failed not only on the first try but also on subsequent attempts. However, these failing tests served as a great opportunity to learn new coding techinques and work alongside the Mitiq team to resolve them. The unwavering support and guidance from the Mitiq team has been instrumental in navigating through these challenges, ensuring that the contribution succedeed. 
+
+## A quick tutorial on using Qibo alongside Mitiq 
+
+### Setup: Defining a circuit using Qibo
+
+For simplicity, we will use a single-qubit circuit with ten Pauli $X$ gates that compiles to the identity, defined below.
```
Comment (nathanshammah) [inline src/content/blog/adding_qibo.md:32]:
```suggestion
For simplicity, we will use a single-qubit circuit with ten Pauli _X_ gates that compiles to the identity, defined below.
```

File: src/content/blog/adding_qibo.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+```python 
+from qibo import Circuit,gates
+
+c = Circuit(1) 
+for _ in range(10): 
+    c.add(gates.X(0))
+c.add(gates.M(0))
+```
+
+In this example, we will use the probability of obtaining the $|0\rangle$ state as our observable to mitigate, the expectation value of which should evaluate to one in the noiseless setting.
```
Comment (nathanshammah) [inline src/content/blog/adding_qibo.md:43]:
```suggestion
In this example, we will use the probability of obtaining the |0⟩ state as our observable to mitigate, the expectation value of which should evaluate to one in the noiseless setting.
```

File: src/content/blog/adding_qibo.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+title: Adding Qibo as a new supported frontend for Mitiq
+author: Francesc Sabater
+day: 24
+month: 2
+year: 2024
+---
+
+In the latest Mitiq release [v0.34.0](https://github.com/unitaryfund/mitiq/discussions/2194), support for [Qibo](https://qibo.science/) has been announced as a newly integrated frontend. This addition expands Mitiq's list of supported frontends, formerly consisting of Cirq, Qiskit, pyQuil, Bracket, and PennyLane, to now include Qibo.
```
Comment (nathanshammah) [inline src/content/blog/adding_qibo.md:9]:
```suggestion
In the latest Mitiq release [v0.34.0](https://github.com/unitaryfund/mitiq/discussions/2194), support for [Qibo](https://qibo.science/) has been announced as a newly integrated frontend. This addition expands Mitiq's list of supported frontends, formerly consisting of Cirq, Qiskit, pyQuil, Braket, and PennyLane, to now include Qibo.
```

File: src/content/blog/adding_qibo.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+title: Adding Qibo as a new supported frontend for Mitiq
+author: Francesc Sabater
+day: 24
```
Comment (nathanshammah) [inline src/content/blog/adding_qibo.md:4]:
```suggestion
day: 29
```

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Hi @francescsabater, this blog post is fantastic. 

I added some small edits. 

Two main improvements: I wanted to link the `from_qibo` conversion from the API-doc and could not find it in the Mitiq docs. I hence opened this PR. Once it's there, we could link it. 

It could be great to add an image to the blog post, at the start, right after the first sentence. [Here](https://docs.google.com/presentation/d/1CNIyBMS2PsGGwKmcX-VPMnZM5jmKyIivpQe_ZyI7pGo/edit?usp=sharing) I tried some options using the Mitiq-Qibo logos and /or also with the code snippet you created, how do you like it? What's best? Maybe just the logos are fine? Cc @francespoblete.

**Comment by francescsabater** [comment]
Hi @nathanshammah , personally, I like the most the one with only the logos. I have added it to the blog post, however it does not show up in the preview.

File: src/content/blog/adding_qibo.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+---
+title: Adding Qibo as a new supported frontend for Mitiq
+author: Francesc Sabater
+day: 29
+month: 2
+year: 2024
+---
+
+In the latest Mitiq release [v0.34.0](https://github.com/unitaryfund/mitiq/discussions/2194), support for [Qibo](https://qibo.science/) has been announced as a newly integrated frontend. This addition expands Mitiq's list of supported frontends, formerly consisting of Cirq, Qiskit, pyQuil, Braket, and PennyLane, to now include Qibo.
+![](../../../public/images/qibo-mitiq.png)
```
Comment (nathanshammah) [inline src/content/blog/adding_qibo.md:10]:
```suggestion
![Qibo Mitiq integration.](/images/qibo-mitiq.png)
```

File: src/content/blog/adding_qibo.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+---
+title: Adding Qibo as a new supported frontend for Mitiq
+author: Francesc Sabater
+day: 29
+month: 2
+year: 2024
+---
+
+In the latest Mitiq release [v0.34.0](https://github.com/unitaryfund/mitiq/discussions/2194), support for [Qibo](https://qibo.science/) has been announced as a newly integrated frontend. This addition expands Mitiq's list of supported frontends, formerly consisting of Cirq, Qiskit, pyQuil, Braket, and PennyLane, to now include Qibo.
```
Comment (nathanshammah) [inline src/content/blog/adding_qibo.md:9]:
```suggestion
**In the latest Mitiq release, [v0.34.0](https://github.com/unitaryfund/mitiq/discussions/2194), support for [Qibo](https://qibo.science/) has been announced as a newly integrated frontend.** 

This addition expands Mitiq's list of supported frontends, formerly consisting of Cirq, Qiskit, pyQuil, Braket, and PennyLane, to now include Qibo.
```

**Review by nathanshammah — COMMENTED** [review]
Excellent, @francescsabater, looks fine now. Merging!

PS: It could be nice to add even a super short tutorial to Mitiq Examples in the doc, it doesn't need to be complex, basically just adding a bit from the snippets you already have.

**Pull request merged**